src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1204,6 +1208,17 @@ static Library()
                 BuiltinFunctionsCore.Pi,
                 NoErrorHandling(Pi)
             },
+            {
+                BuiltinFunctionsCore.PlainText,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.PlainText.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
```

Comment:
>NoOpAlreadyHandledByIR [](http://example.com/codeflow?start=40&length=22)

Should this be `DoNotReplaceBlank` ?

Reply:
I think this is fine. input is blank then output blank. so default let IR handle

Reply:
I think we set this to `NoOpAlreadyHandledByIR` when the IR actually converts Blank to something. Otherwise `DoNotReplaceBlank` is preferred.

Reply:
updated